src/routers/TabRouter.js

```diff
@@ -69,6 +73,9 @@ export default (routeConfigs, config = {}) => {
             params: {
               ...route.params,
               ...params,
+              ...(route.routeName === initialRouteName
+                ? initialRouteParams
+                : {}),
```

Comment:
Can use `null` here instead of empty object

Reply:
Didn't realise this, thanks!